test/Libraries/Microsoft.Extensions.AI.Abstractions.Tests/Microsoft.Extensions.AI.Abstractions.Tests.csproj

```
@@ -6,7 +6,7 @@
 
   <PropertyGroup>
     <NoWarn>$(NoWarn);CA1063;CA1861;CA2201;VSTHRD003;S104</NoWarn>
-    <NoWarn>$(NoWarn);MEAI001</NoWarn>
+    <NoWarn>$(NoWarn);MEAI001;MEAI0001;MEAI0002;MEAI0003;MEAI0004;MEAI0005;MEAI0006;MEAI0007;MEAI0008</NoWarn>
```

The "MEAI001" suppression can now be removed from here and all other previous places since it has been replaced with the more granular suppressions.

```suggestion
    <NoWarn>$(NoWarn);MEAI0001;MEAI0002;MEAI0003;MEAI0004;MEAI0005;MEAI0006;MEAI0007;MEAI0008</NoWarn>
```